docs/site/src/theme/Navbar/Content/index.js

```diff
@@ -1,28 +1,27 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 import React from "react";
-import Link from "@docusaurus/Link";
 import { useThemeConfig, ErrorCauseBoundary } from "@docusaurus/theme-common";
 import {
   splitNavbarItems,
   useNavbarMobileSidebar,
 } from "@docusaurus/theme-common/internal";
 import NavbarItem from "@theme/NavbarItem";
-import NavbarColorModeToggle from "@theme/Navbar/ColorModeToggle";
+import ThemeToggle from "@site/src/components/ThemeToggle";
+//import NavbarColorModeToggle from "@theme/Navbar/ColorModeToggle";
```

Comment:
Similar question here -- how come we are able to replace one component for the other here?

Reply:
Oops. I need to delete that comment. The new components just act as wrappers for those same docusaurus components that show or hide based on location. Reason being is that it's easier to change behavior in this way without changing the default docusaurus component.